src/server/pa_rot.rs

```diff
@@ -245,6 +259,55 @@ mod test {
         Ok(resp_val)
     }
 
+    fn simulate_response<'a, C: protocol::Command<'a>, A: Arena>(
+        scratch_space: &'a mut [u8],
+        arena: &'a mut A,
+        _server: &mut PaRot<fake::Identity, fake::Reset, ring::rsa::Builder>,
+        request: C::Req,
+    ) -> Result<C::Resp, Error> {
+        let len = scratch_space.len();
+        let (req_scratch, _port_scratch) = scratch_space.split_at_mut(len / 2);
```

Comment:
Nit: `_port_scratch` => `_`?

Reply:
I prefer keeping the name here so it's more obvious what it is for future use.